### PR TITLE
bacpop-150 Staging config

### PR DESCRIPTION
**Pull request by EmmaLRussell**
Adds config for beebop-dev.dide.ic.ac.uk

Contains commits from https://github.com/bacpop/beebop-deploy/pull/3 so is pulling in v6 database.

### CODE REVIEW

File: config/prod.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 proxy:
   host: beebop.dide.ic.ac.uk
   ssl:
-    certificate: VAULT:secret/beebop/ssl:cert
-    key: VAULT:secret/beebop/ssl:key
+    certificate: VAULT:secret/beebop/ssl/production:cert
+    key: VAULT:secret/beebop/ssl/production:key
```
Comment (EmmaLRussell) [inline config/prod.yml]: We've added an additional level to keep ssl values for different servers (production and staging).

File: .github/workflows/Pytest.yml [hunk 2]
```diff
@@ -1,12 +1,4 @@
-on:
-  push:
-    branches:
-      - main
-      - master
-  pull_request:
-    branches:
-      - main
-      - master
+on: push
```
Comment (richfitz) [inline .github/workflows/Pytest.yml]: What is this change for?

Comment (EmmaLRussell) [reply .github/workflows/Pytest.yml]: It wasn't building this branch as targeting bacpop-143..

Comment (richfitz) [reply .github/workflows/Pytest.yml]:
```
on:
  push:
    branches:
      - main
      - master
      - "bacpop-*"
  pull_request:
    branches:
      - main
      - master
      - "bacpop-*"
```

I set this pattern up in mcstate2 and it's working well (https://github.com/mrc-ide/mcstate2/commit/d267d0e2cff3d1aa2cc04903966d7b16f0078396)

**Review by richfitz — APPROVED** [review]

**Pull request merged**